SUPPORT.md

```diff
@@ -0,0 +1,3 @@
+ * Ask usage questions on [StackOverflow](https://stackoverflow.com/questions/tagged/cartopy).
```

Comment:
To keep things tidy, I'd advocate moving this to the .github folder (https://github.com/SciTools/cartopy/tree/master/.github). I assume it will continue to work as expected, though that is untested.